rootfs/bin/boot

```diff
@@ -13,62 +13,11 @@ echo system information:
 echo "Django Version: $(./manage.py --version)"
 python --version
 
-# configure etcd
-export ETCD_PORT=${DEIS_ETCD_1_SERVICE_PORT_CLIENT:-4001}
-export ETCD_HOST=${DEIS_ETCD_1_SERVICE_HOST:-$HOST}
-export ETCD="$ETCD_HOST:$ETCD_PORT"
-export ETCD_PATH=${ETCD_PATH:-/deis/controller}
-export ETCD_TTL=${ETCD_TTL:-20}
+# spawn confd in the background to update services based on changes
+confd -backend env --confdir /app --log-level error --interval 5 &
```

Comment:
envvars. `--backend env` switches it from reading etcd to reading from local environment variables: https://github.com/kelseyhightower/confd/blob/master/docs/quick-start-guide.md#environment-variables